.github/workflows/brokenlink-check.yml

```diff
@@ -0,0 +1,15 @@
+name: Links
+
+on:
+  workflow_dispatch:
+
+jobs:
+  my-broken-link-checker:
+    name: Check broken links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
+        uses: ruzickap/action-my-broken-link-checker@v2
+        with:
+          url: https://qiskit.org
+          cmd_params: "--buffer-size=8192 --max-connections=10 --color=always --skip-tls-verification --exclude=\"(https://qiskit.org/documentation|https://qiskit.org/ecosystem/.*)\" --timeout=60"
```

Comment:
Just one question: 

is the check actually excluding links from `/ecosystem/.*`?, becuase it looks like they're still populating in the .csv file... 

or are we only excluding files that live _within_ the `/ecosystem` directory, and _not_ subdirectories?



https://github.com/Qiskit/qiskit.org/assets/6276074/40b4dd99-0d04-44df-b93a-0f934ecc093d

Reply:
Hi @techtolentino, this change was made after this file was generated. I have posted a newer file.

By the way, this workflow is not running yet in this repository (not until we merge this PR). I'm executing it on my own repo where I can execute it every when I need